### DISCUSSION TITLE
Possible to change advertising tx power and channel

### DISCUSSION THREAD

**Discussion by voutti** [ThomasGerstenberg/blatann#95] [Q&A]
I wonder if it is possible to set the advertising power (dBm) ?
Also, is it possible to mask in (use) only some of the three advertising channels (37, 38, 39)?

**Comment by ThomasGerstenberg** [comment]
Looks like this is exposed in the python bindings to `pc-ble-driver` but not currently a part of blatann's api. Feel free to create an enhancement issue, shouldn't be too hard to add in

**Reply by voutti** [reply]
Ok - Thanks Thomas. I added an issue of this.

**Comment by ThomasGerstenberg** [comment]
Added in #98
